### PR TITLE
Docker support

### PR DESCRIPTION
**Pull request by djbios**
Hi! I found your bot very useful, so to be able to deploy it to my server I added the docker support. Please, consider merging, should be useful for others as well

### CODE REVIEW

File: Dockerfile
Comment (dssysolyatin) [inline Dockerfile:1]: I built the image using your Dockerfile, and it takes up 2.5 GB. Please use a multi-stage build to reduce the size to 15 MB.

```
FROM rust:alpine as builder

    # Create a new empty shell project
RUN USER=root cargo new --bin telegram2photoprism && \
    # Dependencies for building openssl
    apk add pkgconfig openssl-dev musl-dev perl make
WORKDIR /telegram2photoprism

# Copy the Cargo.toml and Cargo.lock files
COPY Cargo.toml Cargo.lock ./

# Build the dependencies to cache them
RUN cargo build --release
RUN rm src/*.rs

# Copy the source code
COPY src ./src

# Build the application
RUN cargo build --release

FROM scratch
COPY --from=builder /telegram2photoprism/target/release/telegram2photoprism /
CMD ["/telegram2photoprism"]

```

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ target/
 *.pdb
 .env
 todo.md
+
+.env
```
Comment (dssysolyatin) [inline .gitignore]: The .env is already included in the .gitignore file (line 12). Please revert this change.

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,11 @@
+TELEGRAM2PHOTOPRISM_TELEGRAM_ACCESS_TOKEN=your_telegram_access_token
+TELEGRAM2PHOTOPRISM_TELEGRAM_CHAT_ID=your_telegram_chat_id
+TELEGRAM2PHOTOPRISM_BOT_API_SERVER=your_photoprism_url
```
Comment (dssysolyatin) [inline .env.example]: It is not a good idea to rewrite default parameters inside .env.example. It would be better to write them as a comment:
```
; TELEGRAM2PHOTOPRISM_BOT_API_SERVER=https://api.telegram.org
```

**Review by dssysolyatin — CHANGES_REQUESTED** [review]
Thank you for your PR, @djbios. I have left a few comments. Please address them, and I will merge the PR.

File: .gitignore (hunk 1 above)
Comment (djbios) [reply .gitignore]: Ah, my bad, will remove

**Comment by dssysolyatin** [comment]
@djbios Thanks for your contribution! I will try to add a GitHub action to push the Docker image to Docker Hub over the weekend.

**Comment by dssysolyatin** [comment]
@djbios Finally managed to automate pushing images to Dockerhub. I also made some changes to your Dockerfile because the localization wasn't working, and I added an example of compose files with a Telegram bot server.

- [Docker Image](https://hub.docker.com/r/dssysolyatin/telegram2photoprism)
- [compose.simple.yaml](https://github.com/dssysolyatin/telegram2photoprism/blob/main/compose.simple.yaml): Docker Compose configuration for running telegram2photoprism alone.
- [compose.telegram-bot-server.yaml](https://github.com/dssysolyatin/telegram2photoprism/blob/main/compose.telegram-bot-server.yaml): Docker Compose configuration for running telegram2photoprism with a Telegram bot server.
